playwright/tests/general_regression/sequencing.spec.js

```diff
@@ -4,7 +4,7 @@ require('dotenv').config();
 
 test.use({ storageState: 'tests/state.json' });
 
-test('homepage', async ({ page }) => {
+test('Sequencing add new sequece - edit sequence - remove sequence', async ({ page }) => {
```

Comment:
Small typo. Can we change the title to be something more coherent here? Maybe we could start using a pattern with our titles. 

As it is in sequencing.spec already, I would say this should be something like:

'Add, edit and remove a Sequence'

Reply:
Have updated